#: __init__.py:100
msgid "NextGIS geoinformation system"
msgstr "NextGIS Geoinformationssystem"

#: __init__.py:225
msgid "Linux kernel"
msgstr "Linux-Kern"

#: __init__.py:228
msgid "OS distribution"
msgstr "Betriebssystemverteilung"

#: __init__.py:240
msgid "CPU"
msgstr "Zentralprozessor"

#: __init__.py:244
msgid "RAM"
msgstr "RAM"

#: exception.py:85
msgid "Validation error"
msgstr "Validierungsfehler"

#: exception.py:90
msgid "Forbidden"
msgstr ""

#: exception.py:95
msgid "Insufficient permissions"
msgstr "Berechtigungen nicht ausreichend"

#: exception.py:100
msgid "Operational error"
msgstr "Betriebsfehler"

#: exception.py:105
msgid "External service error"
msgstr ""

#: storage.py:258
msgid "Storage full"
msgstr ""

#: storage.py:259
#, python-format
msgid ""
"This operation can't be performed because the storage limit (%s) has been "
"already exceeded by %s."
msgstr ""

#: storage.py:267
msgid "Not enough storage"
msgstr ""

#: storage.py:268
#, python-format
msgid ""
"This operation requires %s of storage and can't be performed because only %s "
"available."
msgstr ""

#: storage.py:276
msgid "Estimation is already running"
msgstr ""

#: test/test_exception.py:75
#, python-format
msgid "The answer is %d"
msgstr "Die Antwort ist %d"

